src/ram/analysis/IndexAnalysis.h

```diff
@@ -420,6 +361,12 @@ class MinIndexSelection {
     /** @Brief get all chains from the matching */
     const ChainOrderMap getChainsFromMatching(const MaxMatching::Matchings& match, const SearchSet& nodes);
 
+    /** @Brief discharge extra inequalities for a particular existing search given the new search */
```

Comment:
I know this is surrounded by comments starting with `@Brief`, but they're not needed. We configure doxygen to work this out automatically. Capitalise the sentence (also not consistent with some of the surrounding comments, but more consistent overall).

In this case I think it's worth adding `@param` comments for the parameters.